_resources/port1.0/compilers/clang_compilers.tcl

```diff
@@ -7,6 +7,11 @@
 
 global os.major os.platform
 
+# clang is useless on Darwin / PowerPC, let disable it globally
+if {${os.platform} eq "darwin" && [option build_arch] in [list ppc ppc64]} {
```

Comment:
Just to confirm, we want to use `build_arch`, rather than `configure.build_arch`?

Reply:
Indeed, it should be configure.build_arch ...